src/components/Headshots/Headshot.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { faInstagram, faLinkedin, faTiktok, faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { faWindowMaximize } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon as Icon } from "@fortawesome/react-fontawesome";
+import "./Headshot.css";
+
+const Headshot = ({ team, name, role, insta, linkedin, website }) => {
+	const path = `./headshots/${team}/${name}.webp`;
+	const alt = `image of ${name}`;
+	role = role ?? "";
```

Comment:
you can use `role ??= ""` or you can simply put `({ team, name, role = "", instagram, linkedin, website }) => {` on the first line of the function instead.

Also why `insta` instead of `instagram`?